services/QuillLMS/client/app/bundles/Teacher/components/blog_posts/topic_section.jsx

```diff
@@ -44,7 +44,7 @@ export default class TopicSection extends React.Component {
         <div className='meta'>
           <h1>{this.topicIcon()}{this.displayTitle()}</h1>
           <h2>{this.props.articleCount} {pluralize('article', this.props.articleCount)}</h2>
-          <a href={`/${this.sectionLink()}/topic/${this.props.title.toLowerCase().replace(/\s/g, '-')}`}>Show All</a>
+          <a href={`/${this.sectionLink()}/topic/${this.props.title.toLowerCase().replace(/\s/g, '-').replace(/'/g, '%27')}`}>Show All</a>
```

Comment:
Similar to my previous note, we should also be able to extract this from the `TOPIC_SLUGS` value in the controller.  Again, I'm not sure how hard this is to get given our architecture, but I wanted to call it out as it would mean we wouldn't have to come change all of these different bits of code next time there's a change like this.